### PR TITLE
chore: update Vaadin version to 21.0.0, revert V22 changes

### PR DESCRIPTION
**Pull request by web-padawan**
## Description

1. Updated version to 21.0.0 stable
2. Reverted #679 which will land in V22
3. Reverted #692 which will also land in V22
4. Updated some demos to not use TS barrel endpoints feature

## Type of change

- Documentation

### CODE REVIEW

File: articles/fusion/advanced/endpoints-generator.asciidoc [hunk 1]
```diff
@@ -264,10 +264,10 @@ export enum Enumeration {
 
 === Modules
 
-The generator collects all the `_tags_` field of all operations in the OpenAPI document.
-Each tag generates a corresponding TypeScript file.
+The generator will collect all the `_tags_` field of all operations in the OpenAPI document.
```
Comment (jouni) [inline articles/fusion/advanced/endpoints-generator.asciidoc:267]:
```suggestion
The generator collects all the `_tags_` field of all operations in the OpenAPI document.
```

File: articles/fusion/advanced/endpoints-generator.asciidoc [hunk 2]
```diff
@@ -264,10 +264,10 @@ export enum Enumeration {
...
+Each tag will generate a corresponding TypeScript file.
```
Comment (jouni) [inline articles/fusion/advanced/endpoints-generator.asciidoc:268]:
```suggestion
Each tag generates a corresponding TypeScript file.
```

File: articles/fusion/advanced/endpoints-generator.asciidoc [hunk 3]
```diff
@@ -264,10 +264,10 @@ export enum Enumeration {
...
 The tag name is used for TypeScript module name as well as the file name.
-TsDoc of the class is fetched from `_description_` field of the https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#tagObject[tag object] which has the same name as the class.
+TsDoc of the class will be fetched from `_description_` field of the https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#tagObject[tag object] which has the same name as the class.
```
Comment (jouni) [inline articles/fusion/advanced/endpoints-generator.asciidoc:270]:
```suggestion
TsDoc of the class is fetched from `_description_` field of the https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#tagObject[tag object] which has the same name as the class.
```

File: articles/fusion/application/accessing-backend.asciidoc [hunk 4]
```diff
@@ -112,86 +135,29 @@ For more information about supported types, see <<../advanced/type-conversion#,
 == Example TypeScript Module Contents
 
 For example, the generated TypeScript module for the Java endpoint defined in
-<<accessing-backend#how-to-create-vaadin-endpoint,CounterEndpoint.java>> would look as follows:
+ <<accessing-backend#how-to-create-vaadin-endpoint,CounterEndpoint.java>> would look as follows:
```
Comment (jouni) [inline articles/fusion/application/accessing-backend.asciidoc:138]:
```suggestion
<<accessing-backend#how-to-create-vaadin-endpoint,CounterEndpoint.java>> would look as follows:
```

**Review by jouni — APPROVED** [review]
Would be nice to not revert the minor style fixes, so I added some suggestions.

**Comment by web-padawan** [comment]
I don't think we should fix Vale errors as they are not related to this PR 🤷‍♂️

**Comment by jouni** [comment]
> I don't think we should fix Vale errors as they are not related to this PR

Yes, I agree. Hence the `AsciiDoc OK` label 🙂

**Comment by web-padawan** [comment]
There is no preview yet, should we set it up first? Otherwise feel free to merge, or I can do it myself.

**Comment by jouni** [comment]
Preview builds need to be set up explicitly in Netlify. At the moment those are only set up for v14 and latest.

**Comment by web-padawan** [comment]
Thanks for helping with the preview. The build is green so I will merge this PR.

**Pull request merged**